src/drivers/Kconfig

```diff
@@ -7,4 +7,78 @@ config DMIC
 	help
 	  Select for DMIC driver
 
+config DMIC_FIR_LOW_MEMORY
+       bool "Use small FIR tables"
+       default y
+       depends on DMIC
+       select DMIC_FIR_DECIMATE_BY_2
+       select DMIC_FIR_DECIMATE_BY_6
+       help
+		This option is used to minimize driver footprint but
+		preserve support for 48 kHz and 16 kHz sample rates
+		at typical 2.4 MHz microphone clock rate.
+
+config DMIC_FIR_FULL
```

Comment:
I'd name that one something like ADDITIONAL/EXTRA, when I see CONFIG_*_FULL I expect it to have everything when it may not be a case here

Reply:
@singalsu and also include CAVS prefix if Intel only code